docs/statistics/probability/General.md

```diff
@@ -86,4 +176,396 @@ In simpler terms, **probability** is typically used to discuss the likelihood of
+
+<figure markdown="span">
+  ![Correlation Types](/assets/statistics/meme_mutually.jpg){width=50% }
+  <figcaption>(Source: <a href="https://makeameme.org/meme/one-does-not-55c7e12c14">MakeAMeme</a>) </figcaption>
+</figure>
+
+However, some situations, like getting news from multiple sources, do not have mutually exclusive categories (a person can receive news from both TV and the internet). In such cases, probability cannot be computed unless the question is reformulated with exclusive categories.
+
+### Calculation
+
+For discrete random variables, each outcome of an experiment can be assigned a **probability**. The probability of a specific outcome \(X = x\) is calculated using the formula:
+
+???+ defi "Definition: Probability"
+    Probability of a specific outcome
+
+    \[
+    P(X = x) = \frac{\text{Number of favorable outcomes}}{\text{Total number of possible outcomes}}
+    \]
+
+The **probability distribution** of a discrete random variable shows the likelihood of various outcomes occurring. While this distribution helps us understand the chances of different events, it doesn’t allow us to predict the result of any single experiment. However, if the experiment is repeated many times, the overall pattern becomes clearer, following predictable rules.
+
+???+ example "Example: Flip a Coin"
+
+    <iframe src="/assets/statistics/probability_propprob.html" width="100%" height="400px"></iframe>
+    ??? code "Code"
+        ``` py
+        import random
+        import plotly.express as px
+
+        mylist = ["Heads", "Tails"]
+        random.seed(23) # Set seed for reproducibility
+        flips = random.choices(mylist, k=20)
+
+        # Create a histogram
+        fig = px.histogram(x=flips, nbins=2, histnorm='probability')
+
+        # Change bar mode
+        fig.update_traces(marker=dict(color='#00416E', line=dict(color='#00416E', width=0.5)))
+
+        # Set overlay mode
+        fig.update_layout(
+            xaxis_title_text='Result',
+            yaxis_title_text='Probability/Proportion',
+            title=dict(
+                    text=f'<b><span style="font-size: 10pt">Experiment: Flipping 20 Coins </span></b>',
+                ),
+            bargap=0.1,
+            #xaxis_range=[-0.5,1.5])
+
+        for i in range(0, 2):
+            fig.add_shape(
+                type='line',
+                x0=i-0.46,
+                x1=i+0.46,
+                y0=0.5,
+                y1=0.5,
+                line=dict(color='#E87F2B', width=5),
+                xref='x',
+                yref='y',
+                name='Probability' if i == 0 else None,
+                showlegend=(i == 0) 
+            )
+        fig.show()
+        ```
+
+???+ question "Task"
```

Comment:
They should add the probability as lines to all plots from the previous exercise? So you would end up with 6 plots, right? 🤔

Reply:
Basically its one figure. They should use the figure from the example befor and add thos calculated lines.